recipes-bsp/bootfiles/rpi-config.bb

```diff
@@ -209,24 +230,6 @@ do_compile() {
         echo "dtoverlay=${VC4DTBO}" >> $CONFIG
     fi
 
-    # Choose Camera Sensor to be used, default imx219 sensor
-    if [ "${RASPBERRYPI_CAMERA_V2}" = "1" ]; then
```

Comment:
We should make this backwards compatible.

Reply:
At least for a while with a warning.

Reply:
Yes we can add something like:

```
if [ "${RASPBERRYPI_CAMERA_V2}" = "1" ]; then
    # warn...
    RASPBERRYPI_CAMERA="imx..."
fi
```